docroot/modules/custom/va_gov_post_api/src/Service/PostFacilityStatus.php

```diff
@@ -117,6 +109,37 @@ public function queueFacilityStatus(EntityInterface $entity) {
     return $queued_count;
   }
 
+  /**
+   * Adds facility service data to Post API queue by VAMC system.
+   *
+   * @param \Drupal\Core\Entity\EntityInterface $entity
+   *   Entity.
+   *
+   * @return int
+   *   The count of the number of items queued.
+   */
+  public function queueSystemRelatedStatuses(EntityInterface $entity) {
```

Comment:
This also seems misleading as it is queuing more than status.
```suggestion
  public function queueSystemRelatedFacilities(EntityInterface $entity) {
```